src/cmap/connection.ts

```diff
@@ -138,6 +138,7 @@ export interface ConnectionOptions
   tls: boolean;
   noDelay?: boolean;
   socketTimeoutMS?: number;
+  /** @deprecated - Will be removed in a future major release. Users should use AbortSignal instead. */
```

Comment:
This should be marked internal in this PR. Please file a follow up ticket in V7 to potentially release a deprecation in a minor version preceding V7.

Reply:
Thank you!